src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/AttendanceController.java

```diff
@@ -112,6 +112,17 @@ public AttendanceDto postAttendance(
 
     }
 
+    @GetMapping("/{prison}/attendance-for-scheduled-activities")
+    @ApiOperation(value = "Return a set of attendance details for all offenders that have scheduled activity",
+        response = AttendanceDto.class,
```

Comment:
need to put the [response container](http://docs.swagger.io/swagger-core/v1.5.0/apidocs/io/swagger/annotations/ApiOperation.html#responseContainer()) too?